CONTRIBUTING.md

```diff
@@ -57,6 +57,22 @@ Note: `release/x.y` or `hotfix/x.y.z` will be merged back in `master` after a ne
 
 [git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
 
+# Subtree'd library projects
+
+A number of library dependencies are managed as separate open source projects and are git-subtree'd into the WordPress Android app source tree. Use the following command to updated (pull latest) from their respective repos:
+
+        $ git subtree pull --prefix libs/library_name https://github.com/wordpress-mobile/WordPress-Library_Name-Android.git develop
```

Comment:
> My preference.

I'll switch to be more pragmatic here, and because we know more or less who's going to be merging in from the external projects, I'll add the `--squash` :wink: 

> That was true at some point, but we have very few of them, and it serves as an internal documentation more than a guide to external contributors.

Sold! I know I'll be reaching out for that info myself as well so, will add the push command!